deployment/network-operator/README.md

```diff
@@ -561,7 +561,7 @@ optionally deployed components:
 | `nvIpam.repository`       | string | `ghcr.io/mellanox` | NVIDIA IPAM Plugin image repository                                                    |
 | `nvIpam.version`          | string | `v0.0.3`           | NVIDIA IPAM Plugin image version                                                       |
 | `nvIpam.imagePullSecrets` | list   | `[]`               | An optional list of references to secrets to use for pulling any of the Plugin image   |
-| `nvIpam.config`           | string | `''`               | Network pool configuration as described in https://github.com/Mellanox/nvidia-k8s-ipam |
+| `nvIpam.config`           | string | `{"pools": {"rdma-pool": {"subnet": "192.168.0.0/16", "perNodeBlockSize": 100, "gateway": "192.168.0.1"}}}` | Network pool configuration as described in [nvidia-k8s-ipam](https://github.com/Mellanox/nvidia-k8s-ipam), the default defines a single IP Pool named `"rdma-pool"`|
```

Comment:
what do you mean ? 

something like:
`"{\"foo\": \"bar\"}"`

dont u think it will hinder readability ?

also the value above is what we put  in values.yaml as default there are no double quotes there.

Reply:
ack, ill update

Reply:
updated.